node/subsystem/src/messages.rs

```diff
@@ -123,12 +123,11 @@ pub enum CandidateValidationMessage {
 	),
 	/// Validate a candidate with provided, exhaustive parameters for validation.
 	///
-	/// Explicitly provide the `PersistedValidationData` and `ValidationCode` so this can do full
-	/// validation without needing to access the state of the relay-chain. Optionally provide the
-	/// `TransientValidationData` for further checks on the outputs.
+	/// Explicitly provide the `PersistedValidationData`, `TransientValidationData` and `ValidationCode`
+	/// so this can do full validation without needing to access the state of the relay-chain.
```

Comment:
This docs change doesn't make sense, because the only way to access `TransientValidationData` is to access the state of the relay-chain.